Sample sites/jquery/index.html

```diff
@@ -8,11 +8,11 @@
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
     <script>
       var xmpcfg = {
-        access: {
-          accessToken: '8ae058d5-31f0-4e12-8a47-9f82fc6df6ef_399e0b68-6a77-4d2a-b82c-8e6a9eeaa9eb_edbb917d82d64dbd892edca6d0f8246b',
-          url: 'http://52.213.0.23/XMPieXMPL_REST_API',
-          circleProjectID: '399e0b68-6a77-4d2a-b82c-8e6a9eeaa9eb',
-          circleProjectName: 'Rafael XMPL 3'
+        access: { 
+          accessToken: '8ae058d5-31f0-4e12-8a47-9f82fc6df6ef_2d9015c5-f545-4955-9194-8a9253f2bea7_f4a37cbbdc5f482e80a11810de87be2d', 
+          url: 'http://52.213.0.23/XMPieXMPL_REST_API', 
+          circleProjectID:'2d9015c5-f545-4955-9194-8a9253f2bea7', 
+          circleProjectName: 'XMPL SDK Example - New2022 - Login' 
         }
```

Comment:
Could you move outside var xmpcfg = {} to the file xmpcfg.js as we have here https://github.com/XMPieLab/XMPL-NG/wiki/Your-First-Personalized-Page#the-header